float32 goal_velocity
---
bool goal_accepted
